### PR TITLE
shfmt mounts.bats to pass `make validate`

### PR DESCRIPTION
**Pull request by AkihiroSuda**
CI on the master has been broken due to this.

https://travis-ci.org/github/opencontainers/runc/builds/739817745

Not sure why we couldn't catch this before merging it.

### CODE REVIEW

**Review by kolyshkin — APPROVED** [review]
LGTM

**Comment by kolyshkin** [comment]
@opencontainers/runc-maintainers can we please merge this one as otherwise a handful or recent PRs are failing?

**Review by mrunalp — APPROVED** [review]

**Comment by kolyshkin** [comment]
This happened because #2570 was not rebased on top of #2642 after the latter was merged.

**Pull request merged**
